### PR TITLE
New post: "Puppetize Digital 2020 - Content Selection"

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

File: _posts/2020-11-23-puppetize-digital.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  - puppet
+  - conference
+tags:
+  - content
+  - demos
+  - tips-and-tricks
+  - videos
+---
+
+Last week saw [Puppetize Digital 2020](https://digital.puppetize.com/) our annual conference on all things Puppet.
```
Comment (sanfrancrisko) [inline _posts/2020-11-23-puppetize-digital.md:15]:
```suggestion
Last week saw [Puppetize Digital 2020](https://digital.puppetize.com/), our annual conference on all things Puppet.
```

File: _posts/2020-11-23-puppetize-digital.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+There is a [playlist](https://www.youtube.com/playlist?list=PLV86BgbREluXpEPnW_Bb4kgZAlGit4EnW) with all videos, and in this post I'll directly link to my favourites - of course focused on Puppet Content and the IAC team's work.
+
+## Puppet Pair Programming with Visual Studio Code
+
+The [Puppet VSCode Plugin](https://puppet-vscode.github.io/) has evolved into THE canonical way to edit puppet modules.
+The authors of the plugin, [James Pogran](https://www.linkedin.com/in/jamespogran/) and [Glenn Sarti](https://www.linkedin.com/in/glenn-sarti/) are sharing the newest updates to the plugin, as well as some of the benefits of working with a modern, full-featured Development Environment:
+
+{% youtube j3o-U1c_dXs 800 450 %}
+
+Since last week we also provide a default [gitpod](https://gitpod.io) and [codespaces](https://github.com/features/codespaces) environment on our supported modules. So either install the [gitpod browser extension]()
```
Comment (sanfrancrisko) [inline _posts/2020-11-23-puppetize-digital.md:26]: Looks like something is missing here?

File: _posts/2020-11-23-puppetize-digital.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  - conference
+tags:
+  - content
+  - demos
+  - tips-and-tricks
+  - videos
+---
+
+Last week saw [Puppetize Digital 2020](https://digital.puppetize.com/) our annual conference on all things Puppet.
+If you've missed something as it was streamed, over the weekend the production team uploaded all the talks to [the Puppet YouTube channel](https://www.youtube.com/user/PuppetLabsInc/videos).
```
Comment (sanfrancrisko) [inline _posts/2020-11-23-puppetize-digital.md:16]: Slight rephrase suggestion:
```suggestion
Over the weekend, the production team uploaded all the talks to [the Puppet YouTube channel](https://www.youtube.com/user/PuppetLabsInc/videos), so if you missed anything, you can now catch up at your own leisure.
```

File: _posts/2020-11-23-puppetize-digital.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+* announcing the Trusted Contributor programme
+* a demo by [Daniel Carabas](https://www.linkedin.com/in/daniel-carabas-b2bb0851/) to show off our new public Cloud CI for modules
+
+{% youtube FspzumfFvBg 800 450 %}
+
+## Puppetizing DSC
+
+As the IAC's headline feature this year, [Michael Lombardi](https://www.linkedin.com/in/michaeltlombardi/) is talking about how we finally fixed the "DSC-update" problem.
+From now on, we can publish up-to-date puppet wrappers of [all DSC module from the gallery](https://www.powershellgallery.com/packages) (select "DSC Resource" in the filters) to the new [DSC namespace on the forge](https://forge.puppet.com/dsc).
+This means that any improvement on upstream DSC resources will become available in a timely manner - and without additional build steps - to all puppet users.
```
Comment (sanfrancrisko) [inline _posts/2020-11-23-puppetize-digital.md:42]:
```suggestion
This means that any improvement on upstream DSC resources will become available in a timely manner - and without additional build steps - to all Puppet users.
```

File: _posts/2020-11-23-puppetize-digital.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+The way it is built also means that from now on everyone can build their local DSC resources into a puppet module using the [Puppet.Dsc Powershell Module](https://www.powershellgallery.com/packages/Puppet.Dsc).
```
Comment (sanfrancrisko) [inline _posts/2020-11-23-puppetize-digital.md:43]:
```suggestion
The way it is designed also means that from this point forward, everyone can build their local DSC resources into a puppet module using the [Puppet.Dsc Powershell Module](https://www.powershellgallery.com/packages/Puppet.Dsc).
```

**Review by carabasdaniel — APPROVED** [review]

File: _posts/2020-11-23-puppetize-digital.md (hunk 2 above)
Comment (daianamezdrea) [reply _posts/2020-11-23-puppetize-digital.md:26]: I was looking exactly at the same thing on the blog

**Comment by DavidS** [comment]
addressed the comments in #147

**Pull request merged**